src/monitoring_service/constants.py

```diff
@@ -1,2 +1,2 @@
-WAIT_BLOCKS_BEFORE_MONITOR: int = 0
+PERCENT_SETTLE_TIMEOUT_BEFORE_MONITOR: float = 0.3
```

Comment:
"percent" is misleading in this case. This should have a value of `30` for 30% or be called differently (e.g. "ratio" instead of "percent").

Reply:
renamed to `RATIO_OF_SETTLE_TIMEOUT_BEFORE_MONITOR`